includes/Integrations/Site_Kit.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/**
+ * Class Site_Kit
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Analytics;
+use Google\Web_Stories\Story_Post_Type;
+
+/**
+ * Class Site_Kit.
+ */
+class Site_Kit {
+	/**
+	 * Analytics instance.
+	 *
+	 * @var Analytics
+	 */
+	protected $analytics;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Analytics $analytics Analytics instance.
+	 */
+	public function __construct( Analytics $analytics ) {
+		$this->analytics = $analytics;
+	}
+
+	/**
+	 * Initializes all hooks.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return void
+	 */
+	public function init() {
+		add_filter( 'googlesitekit_amp_gtag_opt', [ $this, 'filter_site_kit_gtag_opt' ] );
+
+		if ( in_array( 'analytics', $this->get_site_kit_active_modules_option(), true ) ) {
```

Comment:
This line is repeated 3 times. I think can add a helper method for this. 

How about something like this. 

```
public function is_module_active( $module ) {
     return in_array( $module, $this->get_site_kit_active_modules_option(), true );
}
```
or 

```
public function is_analytics_active() {
     return in_array( 'analytics', $this->get_site_kit_active_modules_option(), true );
}
```